lib/tacos/path_filter.py

```diff
@@ -24,30 +24,36 @@ def match(self, path: str) -> bool:
         return False
 
     @classmethod
-    def from_config(cls, path: OSPath = CONFIG_PATH) -> PathFilter:
+    def from_config(cls, path: OSPath) -> PathFilter:
         """Get a list of allowed globs from a file
 
         Hash comments are removed and blank lines are ignored.
         Inline comments are not allowed
-        An empty or missing file is treated as all allowed.
+        If both the provided and default files are missing, allow all.
         Globs are evaluated with the fnmatch module."""
         lines: list[str] = []
-        # remove from the first unescaped # to the end
         try:
             with path.open() as config:
                 for line in config:
                     line = line.strip()
                     if line and not line.startswith("#"):
                         lines.append(line)
         except FileNotFoundError:
-            pass
+            try:
+                with DEFAULT_PATH.open() as config:
+                    for line in config:
+                        line = line.strip()
+                        if line and not line.startswith("#"):
+                            lines.append(line)
```

Comment:
P2: Use a function twice instead of copy-pasting code, please.


P4: skip it unless you want:
This could become a tidy `config_lines` function, but I'm not sure where it would best live...
Maybe we have enough config-related code that we can justify a new lib.config namespace.